src/docs/product/integrations/gcp-cloud-run/index.mdx

```diff
@@ -1,15 +1,19 @@
-# Google Cloud Run
+---
+title: Google Cloud Run
+sidebar_order: 1
+description: "Learn more about Sentry's Google Cloud Run integration and how it helps developers save time in building and deploying their applications."
+---
 
 Cloud Run helps developers save time in building and deploying their applications. Sentry helps save time in resolving production issues by providing detailed debugging capabilities.
```

Comment:
@ajjindal is the Google Cloud Run integration a project level or organization level integration?

Reply:
Does it need set up once per org OR does it need set up in each project? Or is it an Integration Platform integration?

Reply:
Mimi, it's a project level integration. 

This doesn't show up on our app's integration page but the only place I could find to put our cloud platform integrations.